tests/console/pam.pm

```diff
@@ -38,7 +38,7 @@ sub run {
         my $qa_head_repo = "http://download.suse.de/ibs/QA:/Head/" . 'SLE-' . $version;
         zypper_ar("$qa_head_repo", name => 'qa-head-repo');
     }
-    zypper_call('install bats pam-test pam pam-config snapper');
+    zypper_call('install bats pam-test pam pam-config snapper "group(wheel)"');
```

Comment:
Isn't `system-group-wheel` providing it? I've checked the package and it's provided in SLES, so it's safe change.

Reply:
yes, that's the package system-group-wheel; my fear was basically if we run this test on anythng older, like SLE12 Service Packs. there it might not be present

Reply:
Right, but for SLE 12 it won't work with this code too: https://openqa.suse.de/tests/4399145#step/pam/17
As we don't experience this issue with SLES, I guess we could just limit scope of the change to openSUSE only for now and then address topic of pam tests raised in  https://bugzilla.suse.com/show_bug.cgi?id=1171016 separately. WDYT?